idea/src/org/jetbrains/kotlin/idea/intentions/ChangeVisibilityModifierIntention.kt

```diff
@@ -113,6 +112,7 @@ open class ChangeVisibilityModifierIntention protected constructor(
     class Private : ChangeVisibilityModifierIntention(KtTokens.PRIVATE_KEYWORD), HighPriorityAction {
         override fun applicabilityRange(element: KtDeclaration): TextRange? {
             if (isAnnotationClassPrimaryConstructor(element)) return null
+            if (element is KtProperty && element.hasJvmFieldAnnotation()) return null
```

Comment:
I'd say it's not the best way to fix it. It's more correct to have all necessary checks inside `canBePrivate` and `canBeInternal` (which does not yet exist). Take into account the fact that both `ChangeVisibilityFix` and `ChangeVisibilityModifierIntention` use `canBePrivate` (and should probably share the same logic). Also I'd recommend to consider content of `canBePrivate` inside `MemberVisibilityCanBePrivateInspection` (private fun). I suspect that some checks from there (not all) could be used inside global `canBePrivate` (that can also fix a bunch of minor bugs, not reported yet).